grisera_api/services.py

```diff
@@ -59,120 +59,120 @@ def __init__(self):
 
     def activity_service(self) -> ActivityService:
         if self.persistence_type == PersistenceTypes.GRAPHDB:
-            return ActivityServiceGraphDB()
+            return ActivityServiceGraphDB
```

Comment:
Not that way :facepalm: I though that my comment was pretty clear... You should add parentheses, not remove them...
```python
    def activity_service(self) -> ActivityService:
        if self.persistence_type == PersistenceTypes.GRAPHDB:
            return ActivityServiceGraphDB()
        else:
            return ActivityService()
```